gwcs/wcs.py

```diff
@@ -334,6 +334,61 @@ def _invert(self, *args, **kwargs):
         """
         raise NotImplementedError
 
+    def in_image(self, *args, **kwargs):
+        """
+        Tests if input world coordinate is contained within forward
+        transformation's image and that it maps to the domain of definition
+        of the forward transformation. In practical terms, this function tests
+        that input world coordinate can be converted to input frame and that
+        it is within the forward transformation's ``bounding_box`` when
+        defined.
+
+        Parameters
+        ----------
+        args : float, array like, `~astropy.coordinates.SkyCoord` or
+            `~astropy.units.Unit` coordinates to be inverted
+
+        kwargs : dict
+            keyword arguments to be passed either to ``backward_transform``
+            (when defined) or to the iterative invert method.
+
+        Returns
+        -------
+        result : bool
```

Comment:
boolean or boolean array?

Reply:
I am fixing this. It is both (depending on the input and number of axes in the input frame)